pkg/controller/installation/windows/calico_windows_upgrader.go

```diff
@@ -40,10 +40,12 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+const (
+	defaultMaxUnavailable = 1
```

Comment:
can't this be defaulted to be an int32 with `defaultMaxUnavailable int32 = 1`, then the cast later isn't needed

Reply:
https://github.com/tigera/operator/pull/1804/commits/f940907e37306e04655304bd9d771cc0b96bc1fb